pkg/devfile/parser/types.go

```diff
@@ -3,15 +3,21 @@ package parser
 import (
 	"encoding/json"
 	"fmt"
-	"reflect"
 	"strings"
 
 	v1 "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
 	devfileCtx "github.com/devfile/parser/pkg/devfile/parser/context"
 	"github.com/devfile/parser/pkg/devfile/parser/data"
+	"github.com/pkg/errors"
```

Comment:
shouldn't this file be renamed to something else pkg/devfile/parser/types.go? and also this methods need to be implemented by devfileObj `DevfileObj` ?

Reply:
I agree to rename the file. `types.go` is too confusing
This is how we implement in odo as well, I don't think we want to change the implementation.

Reply:
yes for now lets rename the file.